custom_components/reflex_ag_grid/ag_grid.py

```diff
@@ -370,7 +370,7 @@ def deselectAll(self) -> rx.event.EventSpec:
     def select_rows_by_key(self, keys: list[str]) -> rx.event.EventHandler:
         keys_var = rx.Var.create(keys, _var_is_string=False)
         script = f"""
-let api = refs['{self.get_ref()}']?.current?.api
+let api = {self.api}'];
```

Comment:
```suggestion
let api = {self.api};
```

The tailing quote and square bracket look kind of dodgy